Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsAsset.cs

```diff
@@ -107,6 +117,36 @@ private static void CreateInputActionReferences(InputActionAsset asset)
             }
         }
 
+        internal static void CheckForDefaultUIActionMapChanges()
+        {
+            var asset = GetOrCreate();
+            if (asset != null)
+            {
+                if (s_DefaultUIMap == null)
+                    GetDefaultUIActionMap();
+                
+                var uiMap = asset.actionMaps.FirstOrDefault(m => m.name == "UI");
+                // "UI" action map has been removed or renamed.
+                if (uiMap == null)
+                {
+                    Debug.LogWarning("The default UI action map does not exist.\r\n " +
```

Comment:
Good that there is some feedback to the user about this problem, but the requirement isn't really to have a "default UI action map" right, the requirement is to have "an action map named 'UI' " right?